draft-wicg-http-no-vary-search.md

```diff
@@ -188,6 +189,8 @@ To _parse a URL search variance_ given _value_:
 
 {:aside}
 > In general, this algorithm is strict and tends to return the default URL search variance whenever it sees something it doesn't recognize. This is because the default URL search variance behavior will just cause fewer cache hits, which is an acceptable fallback behavior.
+>
+> However, unrecognized keys at the top level are ignored, to make it easier to extend this specification in the future. To avoid misbehavior with existing client software, such extensions will likely expand, rather than reduce, the set of requests that a cached response can match.
```

Comment:
This is helpful as far as framing goes, but it is a bit of a forward-looking statement.

The pattern I've seen more often is "Search variance MAY contain parameters other than those listed in this specification.  A recipient MUST ignore any parameter that it does not understand."

Reply:
I guess where I struggle is that a strict reading of that would imply that senders can add their own arbitrary data to the header, confident that it will be ignored by recipients -- but in fact in the future those values may become meaningful, so it's unwise for authors to do so.

Tried again to compromise by removing it from the list of authoring requirements proper, but preserving a note that other keys are scary to use.